scripts/start_lisk

```diff
@@ -55,22 +53,22 @@ populate_database() {
 
   if [ "$DATABASE_HOST" == "localhost" ]
   then
-    echo "Looking for database ${DATABASE_NAME}_check"
-    psql -ltAq | grep "^${DATABASE_NAME}_check|"
+    echo "Looking for peers table"
+    psql -qd "$DATABASE_NAME" -c "select count(*) from peers;" &> /dev/null
```

Comment:
A better idea would be to count the number of blocks. If there are more than 2M for example, we can say, there is data enough to not restore it again.

I recommend to use "count estimate" instead of a simple count. Now it is not important but it will avoid us some problems in the future. 

https://wiki.postgresql.org/wiki/Count_estimate